config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/IdentityProvider.java

```diff
@@ -62,6 +66,14 @@ private IdentityProvider(Builder builder) {
     })
     private IdentityProviderDetail identityProviderDetail;
 
+    public String getId() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=121&issues=AWbqThqoTJjFgJLTIz5X&open=AWbqThqoTJjFgJLTIz5X)

[ISSUE_KEY]: AWbqThqoTJjFgJLTIz5X